frontend/src/components/CourseButton.tsx

```diff
@@ -0,0 +1,32 @@
+{/* example use:
+<CourseButton course='COMP1531' disabled={false}/>
+<CourseButton course='COMP1511' disabled={true}/>
+*/}
+import { useState } from 'react';
+
+interface CourseButtonProps {
+    course: string;
+    disabled: boolean;
+}
+const CourseButton: React.FC<CourseButtonProps> = ({ course, disabled }) => {
+    const [isClicked, setIsClicked] = useState(false);
+
+    const handleClick = () => {
+        if (!disabled) {
+            setIsClicked(prevState => !prevState);
+        }   
+    };
+
+    return (
+        <button
+            className={`rounded-full  px-6 py-2  border-2 border-secondary-bg-500 
+            ${disabled ? 'text-tertiary-500 bg-tertiary-bg-500 border-tertiary-bg-500' : 
+            `${isClicked ? 'bg-secondary-bg-500 text-white ' : 'text-secondary-bg-500 bg-white'}} 
```

Comment:
Can remove backticks ` ` and ${} around the isClicked ternary. Since we're already "in" javascript you don't need to use ${} to enter javascript again. Right now the program is reading the javascript code ${....} then panicing

Reply:
okay thank you :)